ts/Stock/StockTools/StockToolbar.ts

```diff
@@ -486,8 +502,28 @@ class Toolbar {
             className: 'highcharts-stocktools-wrapper ' +
                 guiOptions.className + ' ' + bindingsClassName
         });
+
         container.appendChild(wrapper);
 
+        this.showhideBtn = createElement('div', {
+            className: 'highcharts-toggle-toolbar highcharts-arrow-left'
+        }, void 0, wrapper);
+
+        // Toggle menu
+        this.eventsToUnbind.push(
+            addEvent(this.showhideBtn, 'click', (): void => {
+                chart.update({
+                    stockTools: {
+                        gui: {
+                            visible: !self.visible,
+                            placed: true
+                        }
+                    }
+                });
```

Comment:
StockTools has it's update method. Maybe we could use that to avoid some unnecessary logic?

PS: there is no `placed` property in the API. Should it be there?